Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.cpp

```diff
@@ -63,9 +63,27 @@ WebCodecsAudioDecoder::~WebCodecsAudioDecoder()
 {
 }
 
-static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config, const Settings::Values&)
+static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config)
 {
-    return AudioDecoder::isCodecSupported(config.codec);
+    if (StringView(config.codec).trim(isASCIIWhitespace<UChar>).isEmpty())
+        return false;
+
+    if (!config.numberOfChannels)
```

Comment:
sampleRate and numberOfChannels should probably be made required in WebCodecsAudioDecoderConfig.idl instead.

Reply:
Indeed, the initial implementation is incorrect.